v2/webhooks.go

```diff
@@ -0,0 +1,7 @@
+package courier
+
+// WebhookResponse represents the payload provided by an outbound webhook
+type WebhookResponse struct {
+	Data *MessageResponse `json:"data"`
```

Comment:
nit: not sure this should be a pointer for `Data`. Data should always come back populated. Have you seen cases where it was not?

Reply:
I have not. I'm fine with removing that